Tests/UIKitTests/UIViewExtensionsTests.swift

```diff
@@ -252,46 +252,23 @@ final class UIViewExtensionsTests: XCTestCase {
         waitForExpectations(timeout: 0.5)
     }
 
-    func testRotateByAngle() {
+    func testRotateBy() {
         let view1 = UIView()
         let transform1 = CGAffineTransform(rotationAngle: 2)
-        view1.rotate(byAngle: 2, ofType: .radians, animated: false, duration: 0, completion: nil)
+        view1.rotate(by: .init(value: 2, unit: .radians), animated: false, duration: 0, completion: nil)
```

Comment:
I think we never use `.init` in the code. It doesn't look nice.
Also, since this is a test, and not the public library itself, you can use any SwifterSwift extension, like the ones you recently added 😉 
```suggestion
        view1.rotate(by: .radians(2), animated: false, duration: 0, completion: nil)
```